packages/keyring-api/src/btc/types.ts

```diff
@@ -39,30 +69,77 @@ export enum BtcMethod {
   SendBitcoin = 'sendBitcoin',
 }
 
-export const BtcP2wpkhAccountStruct = object({
+const BtcAccountStruct = object({
   ...KeyringAccountStruct.schema,
 
   /**
-   * Account address.
+   * Account supported scopes (CAIP-2 chain ID).
+   */
+  scopes: nonempty(array(enums(Object.values(BtcScope)))),
```

Comment:
Maybe we should keep using normal `CaipChainId` here too? I was thinking of regtests, but I know we already have a "fake" scope for this one (`bip122:regtest`) so maybe that's ok... 

For other kind of accounts, we are just using a `CaipChainId[]`, but that's mainly cause there could be new chains in the future, Bitcoin might be different here 🤔

```suggestion
  scopes: nonempty(array(CaipChainId)),
```

So maybe, for now we could just go with the "general case", WDYT?

Reply:
the `regtest` scope doesn't change, we use `bip122:regtest` because anyone is free to change the genesis block hash but in practice it will never change: https://github.com/bitcoin/bitcoin/blob/v27.0/src/kernel/chainparams.cpp#L476. We can use the "real" scope if you prefer.

We apply the maximum level of validation to addresses, so there is no reason not to do the same for scopes because it's the same argument, we either validate the actual values (which I think is wrong) or we validate the format.
The probability that we want to support new addresses (like `p2ms` or `p2wsh`) is actually higher than the probability of supporting new scopes.

Reply:
moving back to `CaipChainId `